modules/migration-kubernetes-objects.adoc

```diff
@@ -5,9 +5,16 @@
 
 :_content-type: PROCEDURE
 [id="migration-kubernetes-objects_{context}"]
-= Migrating Kubernetes objects
+= Migrating Kubernetes objects using the {mtc-short} API
 
-You can perform a one-time migration of Kubernetes objects that constitute an application's state.
+After you migrate all the PV data, you can use the {mtc-full} ({mtc-short}) API to perform a one-time migration of Kubernetes objects that constitute an application's state.
+
+You do this by configuring MigPlan fields to provide a list of Kubernetes resources with an additional label selector to further filter those resources, and then performing a migration by creating a MigMigration CR. The MigPlan is closed after the migration.
+
```

Comment:
@RichardHoch 
Now that you have altered the options of adding Kubernetes objects to a MigPlan, this explanation is both redundant (addition of Kubernetes objects is explained in the options below, and closing of the MigPlan is stated in the second note below) and wrong (...to further filter...). 
I would suggest to replace this paragraph with:
"To do that, you add the Kubernetes objects that you want to migrate to a `MigPlan` CR and then create a MigMigration CR to migrate them."